FlashFeetAPI/src/main/java/org/example/api/JobRoleService.java

```diff
@@ -0,0 +1,48 @@
+package org.example.api;
+
+import org.example.cli.JobRole;
+import org.example.client.FailedToDeleteJobRoleException;
+import org.example.client.FailedToGetJobsException;
+import org.example.client.JobRoleDoesNotExistException;
+import org.example.db.DatabaseConnector;
+import org.example.db.JobRoleDao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobRoleService {
+
+    private JobRoleDao jobRoleDao = new JobRoleDao();
+
+    public JobRoleService(){
+
+    }
+
+    public JobRoleService(JobRoleDao jobRoleDao, DatabaseConnector databaseConnector) {
+    }
```

Comment:
Is there a reason you need these empty constructors? Especially the second one that ignores the two parameters?